databases/databases-for-redis/how-to/scale-up-a-database-for-redis.mdx

```diff
@@ -0,0 +1,70 @@
+---
+meta:
+  title: How to scale up a Managed Database for Redis™
+  description: This page explains how to scale up a Managed Database for Redis™
+content:
+  h1: How to scale up a Managed Database for Redis™ 
+  paragraph: This page explains how to scale up a Managed Database for Redis™
+dates:
+  validation: 2022-07-11
+  posted: 2022-07-11
+---
+
+If you want to scale up the size of your Database for Redis™, you can edit your node settings to either:
+
+  - **scale vertically**: upgrade to a bigger node type to allow increased traffic for the same endpoint.
+  - **scale horizontally**: add more nodes to your Database Instance to increase your number of endpoints and distribute cache. This setting is only avaialble for Database Instances in cluster mode.
+
+## How to scale vertically 
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have a [Database for Redis™](/databases/databases-for-redis/how-to/create-a-database-for-redis)
+
+</Message>
+
+1. Click **Databases for Redis™** under **Database** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the database information page.
+3. Scroll down the page to the **Node Settings** section.
+4. Click **Scale vertically**. A pop-up appears.
+5. Select a new node type from the drop-down.
+6. Check the details of the estimated cost and click **Validate**. 
+
+  You can view the details of your new node type in the Database Instance information page.
+
+## How to scale horizontally
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have a [Database for Redis™](/databases/databases-for-redis/how-to/create-a-database-for-redis) Instance in [cluster mode](/concepts/#cluster-mode)
+
+</Message>
+
+1. Click **Databases for Redis™** under **Database** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the database information page.
+3. Scroll down the page to the **Node Settings** section.
+4. Click **Scale horizontally**. A pop-up appears.
+5. Set a new total number of nodes. 
+
+  <Message type="note">
+
+  The new total number of nodes has to be bigger than your current number of nodes.
```

Comment:
The minimum number of nodes in a cluster is 3 nodes, you can scale up to 6 nodes